.jshintrc

```diff
@@ -49,7 +49,7 @@
     "lastsemic"     : false,       // Suppresses warnings about missing semicolons, but only when the semicolon is omitted for the last statement in a one-line block.
     "laxbreak"      : false,       // Suppresses most of the warnings about possibly unsafe line breaks in your code.
     "laxcomma"      : false,       // Suppresses warnings about comma-first coding style.
-    "loopfunc"      : false,       // Suppresses warnings about functions inside of loops.
+    "loopfunc"      : true,        // Suppresses warnings about functions inside of loops.
```

Comment:
I think flipping this to `false` is in fact the right idea -- we have a few bugs in `spec_helpers.js` that would have been caught if it were! I'll fix those bugs here, and make a PR against edx-platform as well.